.github/workflows/ci.yml

```
@@ -0,0 +1,265 @@
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+            branch="${{ github.base_ref }}"
+          else
+            branch="${{ github.ref_name }}"
+          fi
+          if [[ "$branch" == "main" ]]; then
+            echo "strict=true" >> "$GITHUB_OUTPUT"
+          else
+            echo "strict=false" >> "$GITHUB_OUTPUT"
+          fi
+      - name: Run showcases
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:5:28: Double quote to prevent globbing and word splitting

(shellcheck)